docs/samples/python/langextract/index.md

```
@@ -159,17 +159,19 @@ To use a custom configuration file:
 
 ```python
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)